### PR TITLE
Feature/custom scripts

### PR DESCRIPTION
**Pull request by jcrater**
Limit exclusion to scripts directly beneath service/bin instead of recursively.

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/sls-packaging/249)

### CODE REVIEW

File: gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/DistTarTask.groovy [hunk 1]
```diff
@@ -51,7 +51,7 @@ class DistTarTask {
 
             from("${project.projectDir}/service") {
                 into "${archiveRootDir}/service"
-                exclude("**/bin/*")
+                exclude("bin/*")
```
Comment (uschi2000) [inline gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/DistTarTask.groovy:54]: can you add a test illustrating what went wrong in https://github.com/palantir/sls-packaging/issues/247 ?

Comment (ash211) [reply gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/tasks/DistTarTask.groovy:54]: The problem I observed was that this folder in my sls-packaged tarball was stripped empty: `spark-history-0.7.0-rc5-1-g52b7787/service/spark/spark-dist-2.3.0-palantir.13-hadoop-2.8.0-palantir.8/bin/`

**Comment by pnepywoda** [comment]
+1
we need this PR to start including `service/monitoring/bin` again (fixing the regression in https://github.com/palantir/sls-packaging/pull/240)

**Comment by pnepywoda** [comment]
ping @uschi2000

**Comment by uschi2000** [comment]
I'm without laptop today, sorry.

Sent from Email+ secured by MobileIron

-------- Original Message --------

**Review by uschi2000 — APPROVED** [review]

**Comment by jcrater** [comment]
Hey, guys. Sorry, I'm high side all day without access to laptop. I'll try to find some time to figure out the gradle testing framework and add a test.

**Pull request merged**